src/components/compoundComponents/Breadcrumbs/BreadcrumbsExample.tsx

```diff
@@ -0,0 +1,17 @@
+import Breadcrumbs from './Breadcrumbs';
+
+function BreadcrumbsExample() {
```

Comment:
Do you want to add more complex examples for this too? I think in this example it might not be clear for our readers why this is so awesome (e.g. could also do this with a prop, there is not state or information shared between the two)

Reply:
Yes. I add more complex example(s).

Reply:
@sebald I added another example.